lsp-ltex.el

```diff
@@ -398,7 +398,9 @@ This file is use to activate the language server."
  (make-lsp-client
   :new-connection (lsp-stdio-connection
                    #'lsp-ltex--server-command
-                   (lambda () (and (file-executable-p (lsp-ltex--server-entry)) (file-exists-p (lsp-ltex--server-entry)))))
+                   (lambda () (and (file-exists-p (lsp-ltex--server-entry))
+                                   (not (file-directory-p (lsp-ltex--server-entry)))
+                                   (file-executable-p (lsp-ltex--server-entry)))))
```

Comment:
Maybe we should wrap it with `let` clause so it save a bit of time.

```elisp
(let ((exec (lsp-ltex--server-entry)))
  ...
  )
```

Reply:
Done!